util/logger.go

```diff
@@ -3,47 +3,60 @@ package util
 import (
 	"fmt"
 	"io"
-	"log"
 	"os"
-	"strings"
 
 	"github.com/fatih/color"
 	"github.com/hashicorp/hcl/v2"
 	"github.com/hashicorp/hcl/v2/hclparse"
+	"github.com/sirupsen/logrus"
 	"golang.org/x/crypto/ssh/terminal"
 )
 
-// Create a logger with the given prefix
-func CreateLogger(prefix string) *log.Logger {
-	return CreateLoggerWithWriter(os.Stderr, prefix)
+// CreateLogger creates a logger. If debug is set, we use ErrorLevel to enable verbose output, otherwise - only errors are shown
+func CreateLogger(lvl string) *logrus.Logger {
+	logger := logrus.New()
+	level, err := logrus.ParseLevel(lvl)
+	if err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
```

Comment:
We try very hard to avoid calling `panic` or `os.Exit` in pretty much all of our code (see our [Go style guide](https://gruntwork.io/guides/style%20guides/golang-style-guide/#error-handling)), as that makes the code easier to test, re-use, etc. This should probably return an error instead.